nss-day-cohort-47/tabloidfullstack-eureka
# list of all categories

# Description
list of categories pulled from the database shown on the DOM
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Once logged in, click on the categories link in the navbar. There should be a list of categories on the page.
# Checklist:
